src/openlcb/MultiConfiguredPC.hxx

```diff
@@ -69,7 +69,7 @@ CDI_GROUP_END();
 
 /// CDI Configuration for a @ref ConfiguredConsumer.
 CDI_GROUP(PCConfig);
-enum class ActionConfig : uint8_t
+enum class ActionConfig : uint16_t
```

Comment:
I don't see a reason for this change. Could you explain why it is needed?
There are only two values.